atomic_defi_design/qml/Screens/FirstLaunch.qml

```diff
@@ -336,8 +336,8 @@ SetupPage {
                                 onClicked: {
                                     let wallet_name = model.modelData
                                     let dialog = app.getText({
-                                        "title": qsTr("Delete Wallet"),
-                                        text: qsTr("Do you want to delete this wallet"),
+                                        "title": qsTr("Delete") + " %1 ".arg(wallet_name) + ("wallet?"),
+                                        text: qsTr("Enter password to confirm wallet delete"),
```

Comment:
"Enter password to confirm the deletion of the wallet" is probably more accurate, cc @smk762  if you have any other idea ?

Reply:
yeah, that's better :+1:
Could reduce to `Enter password to confirm deletion of {wallet_name} wallet`